lib/screens/study_nurse_screen/configuration_screen/pages/configuration_form.dart

```diff
@@ -154,12 +155,17 @@ class _ConfigurationFormState extends State<ConfigurationForm> {
             onChanged: (String? newValue) {
               widget.callbackForm('entryEmail', newValue!);
             },
-            validator: qValidator(
-              <TextValidationRule>[
-                IsRequired(AppLocalizations.of(context)!.entryEmail),
-                IsEmail(AppLocalizations.of(context)!.entryValidEmail),
-              ],
-            ),
+            validator: (String? value) {
+              if (value == null || value.isEmpty) {
+                return AppLocalizations.of(context)!.entryEmail;
+              } else if (!RegExp(
+                r"^[a-zA-Z0-9.a-zA-Z0-9.!#$%&'*+-/=?^_`{|}~]+@[a-zA-Z0-9]+\.[a-zA-Z]+",
+              ).hasMatch(value)) {
+                return AppLocalizations.of(context)!.entryValidEmail;
+              } else {
+                return null;
+              }
```

Comment:
```suggestion
              if (value == null || value.isEmpty) {
                return AppLocalizations.of(context)!.entryEmail;
              } else if (!RegExp('.+@.+',).hasMatch(value)) {
                return AppLocalizations.of(context)!.entryValidEmail;
              } else {
                return null;
              }
```
Regex is too difficult and doesn't even include some valid cases.
A simple regex as that should suffice.